### DISCUSSION TITLE
文件夹加密问题

### DISCUSSION THREAD

**Discussion by Ravenbaar** [AlistGo/alist#1184] [Q&A]
各位大佬好，有一个问题想要请教一下
exp：/a 已加密 但我知道a目录下有一个b目录或者文件，然后我直接访问/a/b即可越过密码验证直接访问/a/b里的东西
这样能修复嘛？

**Comment by anwen-anyi** [comment]
检查父目录密码打开即可~

**Reply by alimalls** [reply]
> 检查父目录密码打开即可~

V3这个现象还是存在的。新版没看到，检查父目录密码选项。

**Reply by anwen-anyi** [reply]
没看到 应用到子文件夹吗 QAQ

**Reply by alimalls** [reply]
> 没看到 应用到子文件夹吗 QAQ

就是勾了应用到子文件夹也不起作用。

**Comment by sjkhsl** [comment]
> 

你好，怎么给文件夹加密？

**Reply by Pandaft** [reply]
仔细查看文档，在 `元信息` 中
